caper/cromwell_metadata.py

```diff
@@ -295,14 +303,22 @@ def gcp_monitor(
                 ...
             ]
         """
+        import pandas as pd
+
```

Comment:
Would prefer to keep this at the top of the file.

Reply:
Done as you suggested.